exe/Main.hs

```diff
@@ -42,9 +46,23 @@ main = do
             _ -> (Info, Nothing, False)
 
     withDefaultRecorder logFilePath Nothing minPriority $ \textWithPriorityRecorder -> do
-      let recorder =
-            textWithPriorityRecorder
-            & cfilter (\WithPriority{ priority } -> priority >= minPriority)
-            & cmapWithPrio pretty
+      let
+        recorder = cmapWithPrio pretty $ mconcat
+            [textWithPriorityRecorder
+                & cfilter (\WithPriority{ priority } -> priority >= minPriority)
+            , lspRecorder
+                & cfilter (\WithPriority{ priority } -> priority >= Error)
+                & cmapWithPrio renderDoc
+            ]
+        plugins = Plugins.idePlugins (cmapWithPrio LogPlugins recorder) includeExamplePlugins
 
-      defaultMain (cmapWithPrio LogIdeMain recorder) args (Plugins.idePlugins (cmapWithPrio LogPlugins recorder) includeExamplePlugins)
+      defaultMain (cmapWithPrio LogIdeMain recorder) args (pluginDescToIdePlugins [lspRecorderPlugin] <> plugins)
+
+renderDoc :: Doc a -> Text
+renderDoc d = renderStrict $ layoutPretty defaultLayoutOptions $ vcat
+    ["Unhandled exception, please check your setup and/or the [issue tracker](" <> issueTrackerUrl <> "): "
```

Comment:
It's not necessarily an exception, right? It could be any other error.

Reply:
True, I copied this message over from an earlier exception site - will change to error

Reply:
And indeed, since we send it as an error message, we don't need to tell the user again that it's an error. Including the issue link is maybe still hepful.